WpfAnimatedGif/ImageBehavior.cs

```diff
@@ -237,8 +237,8 @@ public static void RemoveAnimationLoadedHandler(Image image, RoutedEventHandler
             EventManager.RegisterRoutedEvent(
                 "AnimationCompleted",
                 RoutingStrategy.Bubble,
-                typeof (RoutedEventHandler),
-                typeof (ImageBehavior));
+                typeof(RoutedEventHandler),
+                typeof(ImageBehavior));

```

Comment:
Minor issue: please try not to change formatting of unrelated code.
This is a general rule for open-source contributions, not just for this project. It adds noise to the diff, and the project maintainer might not have the same formatting rules as you.
I find the `git add -a` command (interactive add) useful for staging only specific lines in a file. Many GUI tools support this too (SourceTree, GitExtensions, GitKraken...)